crates/bin/pindexer/src/indexer_ext.rs

```diff
@@ -7,8 +7,8 @@ impl IndexerExt for cometindex::Indexer {
         self.with_index(crate::shielded_pool::fmd::ClueSet {})
             .with_index(crate::stake::ValidatorSet {})
             .with_index(crate::stake::Slashings {})
-            .with_index(crate::stake::MissedBlocks {})
```

Comment:
Why remove `MissedBlocks` here?

Reply:
We don't consume this in our dashboards right now, and this slows down indexing a lot, since this is a frequent event.